x-pack/plugins/ingest/index.ts

```diff
@@ -47,6 +47,14 @@ export function ingest(kibana: any) {
           title: 'Logs',
           url: `/app/${APP_ID}#/logs`,
         },
+        {
+          description: 'Security Operations',
+          euiIconType: 'securityApp',
+          id: 'secops:home',
+          order: 8001,
```

Comment:
What does the key of `order` do? Does it matter that it's the same number as `Explore your logs` above?

Reply:
oups